index.js

```diff
@@ -161,8 +160,10 @@ function writeToDb(statement, values) {
   const db = createDbClient();
   db.query(statement, values, (err, res) => {
     if (err) throw err;
```

Comment:
This seemed like a slight security risk, so I'm making it only happen in verbose mode.